stan/math/prim/mat/fun/gp_exp_quad_cov.hpp

```diff
@@ -34,28 +38,28 @@ template <typename T_x, typename T_sigma, typename T_l>
 inline
     typename Eigen::Matrix<typename stan::return_type<T_x, T_sigma, T_l>::type,
                            Eigen::Dynamic, Eigen::Dynamic>
-    gp_exp_quad_cov(const std::vector<T_x>& x, const T_sigma& sigma,
-                    const T_l& length_scale) {
+    gp_exp_quad_cov(const std::vector<T_x> &x, const T_sigma &sigma,
+                    const T_l &length_scale) {
   using std::exp;
-  check_positive("gp_exp_quad_cov", "marginal variance", sigma);
-  check_positive("gp_exp_quad_cov", "length-scale", length_scale);
+  check_positive("gp_exp_quad_cov", "magnitude", sigma);
```

Comment:
Motivation for the name change? It's not listed anywhere on the PR. It is a side effect that should be listed in the PR and should propagate to the issue itself. (It's a side effect because it actually does change what the user sees for an error message. It's not big, but I've now spent a few minutes in this review thinking about why this is now "magnitude" instead of "marginal variance.")

Reply:
This is consistent with GMPL (a common reference/handbook for GP people). Stan org uses "marginal variance". If there's an objection, happy to change it back

Reply:
No objection. But please document it in the issue / PR.

Reply:
What do you mean "Stan org"?

Reply:
got it. this organization and associated writings

Reply:
really didn't mean anything by it, I just hadn't seen it anywhere else. Then again, I've only read some small subset of the literature so I could be totally wrong. I guess marginal variance is more easily interpreted by statistics people than ML people.